cusub_cortex/state_machine/config/mission_config_sim.yaml

```diff
@@ -35,4 +43,4 @@ tasks:
     roulette: roulette_watershed
 
 mission_tasks: # Tasks To Do In Order
- - start_gate
```

Comment:
Why not start gate?

Reply:
Actually this file no longer exists. There is a mission_config_sim.yaml and mission_config_sub.yaml so that we can store sub specific and sim specific parameters such as imu bump detection axis and thresholds.